libafl/src/events/launcher.rs

```diff
@@ -769,12 +772,16 @@ where
             let centralized_hooks = tuple_list!(CentralizedLlmpHook::<S::Input>::new()?);
 
             // TODO switch to false after solving the bug
-            LlmpBroker::with_keep_pages_attach_to_tcp(
+            let mut broker = LlmpBroker::with_keep_pages_attach_to_tcp(
                 self.shmem_provider.clone(),
                 centralized_hooks,
                 self.centralized_broker_port,
                 true,
-            )?
+            )?;
+            broker
+                .inner_mut()
```

Comment:
Why do we have to access the `inner` here? It should be forwarded, I guess?

Reply:
This was the existing api but I will add a `set_exit_after` to `Broker`